bitcoin/src/blockdata/script/owned.rs

```diff
@@ -155,6 +158,14 @@ crate::internal_macros::define_extension_trait! {
     }
 }
 
+#[cfg(feature = "arbitrary")]
+impl<'a> Arbitrary<'a> for ScriptBuf {
```

Comment:
Oh wow, thanks to the lifetime we can also implement `Arbitrary` on `&'a Script` by calling `u.bytes(len)?`. That's really cool!